components/brave_ads/core/internal/processors/contextual/text_embedding/text_embedding_processor.cc

```diff
@@ -35,7 +35,7 @@ void TextEmbeddingProcessor::Process(const std::string& html) {
     return;
   }
 
-  const std::string text = SanitizeHtml(html);
+  const std::string text = rust::ParseAndSanitizeOgTagAttribute(html);
```

Comment:
Currently, we match the following:
`<... property="og:title" content="some text" ...>`

If we pass the tag and attribute values we will match this:
`<... property="$tag_value" $attribute_value="some text" ...>`

So, I'm not sure if this could be reused.